roles/import_image_and_run_aws_instance/README.md

```diff
@@ -0,0 +1,118 @@
+* kms:CreateGrant
+* kms:Decrypt
+* kms:DescribeKey
+* kms:Encrypt
+* kms:GenerateDataKey*
+* kms:ReEncrypt*
+
+Role Variables
+--------------
+
+* **import_image_and_run_aws_instance_import_image_task_name**: (Required) The name you want to assign to the AWS EC2 import image task.
+* **import_image_and_run_aws_instance_bucket_name**: (Required) The name of the S3 bucket name where you want to upload the .raw image.
+**import_image_and_run_aws_instance_image_path**: (Required) The path where the .raw image is stored.
+* **import_image_and_run_aws_instance_instance_name**: (Required) The name of the EC2 instance you want to create using the imported AMI.
+* **import_image_and_run_aws_instance_instance_type**: The EC2 instance type you want to use. Default: "t2.micro".
+* **import_image_and_run_aws_instances_keypair_name**: The name of the SSH access key to assign to the EC2 instance. It must exist in the region the instance is created. If not set, your default AWS account keypair will be used.
+* **import_image_and_run_aws_instance_security_groups**: A list of security group IDs or names to associate to the EC2 instance.
+* **import_image_and_run_aws_instance_vpc_subnet_id**: The subnet ID in which to launch the EC2 instance (VPC). If none is provided, M(amazon.aws.ec2_instance) will choose the default zone of the default VPC.
+**import_image_and_run_aws_instance_volumes** (dict): (Optional) A dictionary of a block device mappings, by default this will always use the AMI root device so the **instance_volumes** option is primarily for adding more storage. A mapping contains the (optional) keys:
+    * **device_name** (str): The device name (for example, /dev/sdh or xvdh).
+    * **ebs** (dict): Parameters used to automatically set up EBS volumes when the instance is launched.
+        * **volume_type** (str): The volume type. Valid Values: standard, io1, io2, gp2, sc1, st1, gp3.
+        * **volume_size** (int): The size of the volume, in GiBs.
+        * **kms_key_id** (str): Identifier (key ID, key alias, ID ARN, or alias ARN) for a customer managed CMK under which the EBS volume is encrypted.
+        * **iops** (str): The number of I/O operations per second (IOPS). For gp3, io1, and io2 volumes, this represents the number of IOPS that are provisioned for the volume. For gp2 volumes, this represents the baseline performance of the volume and the rate at which the volume accumulates I/O credits for bursting.
+        * **delete_on_termination_** (bool): Indicates whether the EBS volume is deleted on instance termination.
+
+Dependencies
+------------
+
+- role: [aws_setup_credentials](../aws_setup_credentials/README.md)
+
+Example Playbook
+----------------
+This role can be used together with the [cloud.aws_ops.clone_on_prem_vm](../clone_on_prem_vm/README.md) role as shown below. If you wish to use it separately, just ensure you set the **clone_on_prem_vm_local_image_path** parameter.
```

Comment:
I'm not quite sure I follow this. If using this role on its own, wouldn't they just need to set the required `import_image_and_run_aws_instance_image_path` variable?

Reply:
I removed that phrase because it feels to me that it brings more confusion. It's clear that the roles can be used separately and what needs to be specified.